Rubberduck.Inspections/QuickFixes/ChangeIntegerToLongQuickFix.cs

```diff
@@ -0,0 +1,210 @@
+            {
+                ReplaceTypeHint(result.Context, rewriter);
+            }
+            else
+            {
+                switch (result.Target.DeclarationType)
+                {
+                    case DeclarationType.Variable:
+                        var variableContext = (VBAParser.VariableSubStmtContext)result.Context;
+                        rewriter.Replace(variableContext.asTypeClause().type(), Tokens.Long);
+                        break;
+                    case DeclarationType.Constant:
+                        var constantContext = (VBAParser.ConstSubStmtContext)result.Context;
+                        rewriter.Replace(constantContext.asTypeClause().type(), Tokens.Long);
+                        break;
+                    case DeclarationType.Parameter:
+                        var parameterContext = (VBAParser.ArgContext)result.Context;
+                        rewriter.Replace(parameterContext.asTypeClause().type(), Tokens.Long);
+                        break;
+                    case DeclarationType.Function:
+                        var functionContext = (VBAParser.FunctionStmtContext)result.Context;
+                        rewriter.Replace(functionContext.asTypeClause().type(), Tokens.Long);
+                        break;
+                    case DeclarationType.PropertyGet:
+                        var propertyContext = (VBAParser.PropertyGetStmtContext)result.Context;
+                        rewriter.Replace(propertyContext.asTypeClause().type(), Tokens.Long);
+                        break;
+                    case DeclarationType.UserDefinedTypeMember:
+                        var userDefinedTypeMemberContext = (VBAParser.UdtMemberContext)result.Context;
+                        if (userDefinedTypeMemberContext.reservedNameMemberDeclaration() != null)
+                        {
+                            rewriter.Replace(
+                                userDefinedTypeMemberContext.reservedNameMemberDeclaration().asTypeClause().type(),
+                                Tokens.Long);
+
+                        }
+                        else
+                        {
+                            rewriter.Replace(
+                                userDefinedTypeMemberContext.untypedNameMemberDeclaration().optionalArrayClause().asTypeClause().type(),
+                                Tokens.Long);
+                        }
+                        break;
+                }
+            }
+
+            var interfaceMembers = _state.DeclarationFinder.FindAllInterfaceMembers().ToArray();
+
+            ParserRuleContext matchingInterfaceMemberContext;
+
+            switch (result.Target.DeclarationType)
+            {
+                case DeclarationType.Parameter:
+                    matchingInterfaceMemberContext =
+                        interfaceMembers.Select(member => member.Context)
+                            .FirstOrDefault(c => c == result.Context.Parent.Parent);
```

Comment:
why the linebreaks here? Also `result.Context.Parent.Parent` seems very susceptible to NREs